### DISCUSSION TITLE
86Box Auto Updater

### DISCUSSION THREAD

**Discussion by Slo2018** [86Box/86Box#3342] [Ideas]
Please, add Auto Updater in 86Box.

**Comment by OBattler** [comment]
I'm quite sure this is planned as part of the built-in manager that's being worked on.

**Comment by jadasse** [comment]
> I'm quite sure this is planned as part of the built-in manager that's being worked on.

https://github.com/86Box/86BoxManager/issues/90 yep

**Comment by OBattler** [comment]
No, that's not the planned built-in manager, but the old external manager by daviunic.
